specs/CalendarView/CalendarViewSpec1.md

```diff
@@ -0,0 +1,498 @@
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+## Today styling
+The following properties of type Windows.UI.Xaml.Media.Brush are used to customize the current day's rendering when CalendarView.IsTodayHighlighted is True:
+TodayBackground, TodayBlackoutBackground, TodayBlackoutForeground, TodayHoverBackground, TodayPressedBackground, TodaySelectedInnerBorderBrush.
+
+### Showcasing the TodayBackground property in the month, year and decade views
+![Default Today Background.](images/DefaultTodayBackground.png)
+![Default Today Month Background.](images/TodayMonthBackground.png)
+![Default Today Year Background.](images/TodayYearBackground.png)
+
+### Showcasing the TodayHoverBackground property - current date is hovered
+![Hover Today Background.](images/HoverTodayBackground.png)
+
+### Showcasing the TodayPressedBackground property - current date is pressed
+![Pressed Today Background.](images/PressedTodayBackground.png)
+
+### Showcasing the TodayBlackoutBackground and TodayBlackoutForeground properties - the old property CalendarView.IsTodayBlackedOut is True
+![Blacked Out Today Background And Foreground.](images/BlackoutTodayBackgroundAndForeground.png)
+
+### Showcasing the TodaySelectedInnerBorderBrush property - current date is selected
+![Selected Today Border.](images/SelectedTodayBorder.png)
+
+## Blacked Out styling
+The following properties of type Windows.UI.Xaml.Media.Brush are used to customize the calendar items that are blacked out in the month view:
+TodayBlackoutBackground, TodayBlackoutForeground, BlackoutStrikethroughBrush.
+
+### Showcasing the BlackoutStrikethroughBrush property - all Sundays are blacked out
+![Blacked Out Sundays.](images/BlackoutStrikethroughBrush.png)
+
+## Out Of Scope styling
+The following properties of type Windows.UI.Xaml.Media.Brush are used to customize the calendar items that are outside the current scope (month, year, or decade):
+OutOfScopeHoverForeground, OutOfScopePressedForeground.
+
+### Showcasing the OutOfScopeHoverForeground property - 4th of May is hovered
+![Hover Out Of Scope Foreground.](images/HoverOutOfScopeForeground.png)
+
+### Showcasing the OutOfScopePressedForeground property - 4th of May is pressed
+![Pressed Out Of Scope Foreground.](images/PressedOutOfScopeForeground.png)
+
+## Item background styling
+The following properties of type Windows.UI.Xaml.Media.Brush are used to customize the background rendering of items in any DisplayMode:
+CalendarItemHoverBackground, CalendarItemPressedBackground.
```

Comment:
So to set both background and foreground of an out-of-scope item I should set both `CalendarItemHoverBackground` and 'OutOfScopeHoverForeground`?

Reply:
Yes. There is no OutOfScopeHoverBackground property. I did not create a new property for every possible situation and focused on fulfilling the designers' Figma requirements.

Reply:
There are no conflicting HoverBackground properties.